README.md

```diff
@@ -185,6 +185,24 @@ require('apostrophe')({
 });
 ```
 
+#### Add `robots.txt` to Your Site
+By default, the SEO extension will add a route to your site for `/robots.txt` that will return a string that allows for all search engines to index your site.
+
+```
+User-agent: *\nDisallow:
+```
+Within the global configuration you can choose to change this to disallow search engine indexing:
+
+```
+User-agent: *\nDisallow: /
+```
+
+You can also elect to select to add a custom string for your `robots.txt`. This can allow you finer control over what sections of your site can be indexed.
```

Comment:
"elect to select" is assonant, but also redundant 😄

Reply:
... and by which bots.

Reply:
Changed.